src/app/clusters/identify-server/identify-server.h

```diff
@@ -58,5 +58,21 @@ struct Identify
     EmberAfIdentifyEffectIdentifier mCurrentEffectIdentifier;
     EmberAfIdentifyEffectIdentifier mTargetEffectIdentifier;
     uint8_t mEffectVariant;
-    bool mActive = false;
+    bool mActive             = false;
+    Identify * next_identify = nullptr;
+
+    bool hasNext()
+    {
+        bool hasNext = false;
```

Comment:
return `this->next_identify != nullptr`?

Reply:
done